### PR TITLE
feat(DDI-1148): Update two column layout doc

### PR DESCRIPTION
**Pull request by Radha-Alberta**
Added the descriptions for examples in two column layout doc

### CODE REVIEW

File: libs/docs/src/components/common/TwoColumnLayout.stories.mdx [hunk 1]
```diff
@@ -62,6 +59,8 @@ import {
 
 #### Basic
 
+For situations where the default values of 256px for the nav column and 100% max-content width are desirable.
```
Comment (ArakTaiRoth) [inline libs/docs/src/components/common/TwoColumnLayout.stories.mdx:62]: I would use a GoACallout component, type information for displaying this text, and all the other new text in this document

Comment (Radha-Alberta) [reply libs/docs/src/components/common/TwoColumnLayout.stories.mdx]: Added description in the callout

**Review by chrisolsen — APPROVED** [review]

**Review by ArakTaiRoth — APPROVED** [review]

**Pull request merged**